config.js

```diff
@@ -23,8 +23,8 @@ config.redis.password = '';
 
 config.cacheTTL = 20; // Time in seconds to store cache in Redis
 
-config.logFile = '/dev/stdout';
-config.logLevel = 1;
+config.log.file = '/dev/stdout';
+config.log.level = 'debug';
```

Comment:
Default for production should be `info`, please also add `config.log.enabled` that can be set to `true/false` - will be easier for users to disable logging (not everybody is familiar with `/dev/null`).

Also descriptions should be added, something like:
```
// Logging
config.log.enabled = true;          // Collect logs (true - enabled, false - disabled)
config.log.file    = '/dev/stdout'; // Output for logs - can be device file or ordinary path
config.log.level   = 'debug';       // Log level - (trace, debug, info, warn, error)
```